packages/components/src/CollapsiblePanel/CollapsiblePanel.snapshot.test.js

```diff
@@ -150,17 +144,30 @@ describe('CollapsiblePanel', () => {
 
 	it('should render themed with textual content', () => {
 		// when
-		const wrapper = renderer.create(<CollapsiblePanel {...propsDescriptivePanel} />)
-			.toJSON();
+		const wrapper = renderer.create(<CollapsiblePanel {...propsDescriptivePanel} />).toJSON();
 
 		// then
 		expect(wrapper).toMatchSnapshot();
 	});
 
 	it('should render themed without textual content', () => {
 		// when
-		const wrapper = renderer.create(
-			<CollapsiblePanel {...propsDescriptivePanelWithoutContent} content={null} />)
+		const wrapper = renderer
+			.create(<CollapsiblePanel {...propsDescriptivePanelWithoutContent} content={null} />)
+			.toJSON();
+
+		// then
+		expect(wrapper).toMatchSnapshot();
+	});
+
+	it('should render custom content', () => {
+		// when
+		const wrapper = renderer
+			.create(
+				<CollapsiblePanel {...props} content={null}>
+					<h2>Custom Content</h2>
+				</CollapsiblePanel>,
+			)
```

Comment:
please don't create new snapshot, no body take care of reading them and validate them.
Could you please add a find in the dom and be sure you find the custom content ?